docs/api.md

```diff
@@ -43,6 +43,7 @@ Generate a response for a given prompt with a provided model. This is a streamin
 
 Advanced parameters (optional):
 
+- `keep_alive`: how long to keep the model in memory. If negative (e.g. `-1s`) the model will stay loaded. If omitted the model will stay loaded for 5 minutes
```

Comment:
For the docs, we may want to only merge this commit once it's in an active release. Or we can flag it here as "upcoming"

Reply:
I can just pull the docs from this PR and then add them back once the next version is out.